src/wp-includes/blocks.php

```diff
@@ -182,7 +182,10 @@ function register_block_style_handle( $metadata, $field_name, $index = 0 ) {
 		return false;
 	}
 
-	$wpinc_path_norm = wp_normalize_path( realpath( ABSPATH . WPINC ) );
+	static $wpinc_path_norm = null;
```

Comment:
@aristath can we use and empty instead of null? 


```suggestion
	static $wpinc_path_norm = '';
```

Reply:
Indeed, that appears to be more consistent with [r54327](https://core.trac.wordpress.org/changeset/54327).